src/frontend/sharedComponents/AddButton.tsx

```diff
@@ -28,7 +28,6 @@ export const AddButton = React.memo(AddButtonNoMemo);
 const styles = StyleSheet.create({
   container: {
     position: "absolute",
-    zIndex: 10,
```

Comment:
Was showing up over the bg map bottom sheet

Reply:
I checked those things, and as far as I know, there aren't any other things that it could be hidden under